docs/reference/mpyfiles.rst

```diff
@@ -86,7 +86,8 @@ and .mpy version.
 =================== ============
 MicroPython release .mpy version
 =================== ============
-v1.19 and up        6
+v1.20 and up        6.1
+v1.19               6
```

Comment:
It's also valid in v1.19.1, so maybe write `v1.19 - v1.19.1`, or `v1.19.x`.

Reply:
done